env/file_system_tracer.h

```diff
@@ -80,6 +79,9 @@ class FileSystemPtr {
     }
   }
 
+  /* Returns the underlying File System pointer */
+  FileSystem* get() const { return fs_.get(); }
```

Comment:
This should be conditional on whether tracing is on or off. If its on, it should return ```fs_tracer_.get()```.